abnf/distributive.txt

```diff
@@ -0,0 +1,15 @@
+ALPHA = %x41-5A / %x61-7A
+COMMA = ","
+
+odataIdentifier= ALPHA *ALPHA
+entitySetName = odataIdentifier
+keyPredicate = "('DEFAULT')"
+entityColNavigationProperty = odataIdentifier
+
+recHierReference = ( entitySetName keyPredicate "/" entityColNavigationProperty
+                   / entitySetName 
+                   ) COMMA
+
+recHierReference2 = ( entitySetName COMMA
+                    / entitySetName keyPredicate "/" entityColNavigationProperty COMMA 
```

Comment:
Here the trailing COMMA within the alternatives ensures that the first alternative does not match, so the parser tries the second alternative.